### PR TITLE
Fix open source builds

### PR DESCRIPTION
**Pull request by bootstraponline**

### CODE REVIEW

File: open_source.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -ex
+
+# Open source users: Run this script before building.
```
Comment (CalvinKern) [inline open_source.sh]: Is this going to be clear to people to run this or should we include a note about this in the readme?

File: open_source_data/soseedygrpc/files.list [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Private info for iOS data seeding server
```
Comment (joehoag) [inline open_source_data/soseedygrpc/files.list]: Should this say "Android" rather than "iOS"?

File: open_source.sh (hunk 1 above)
Comment (bootstraponline) [reply open_source.sh]: That's a good point, I'll updated the readme.

File: open_source_data/soseedygrpc/files.list (hunk 2 above)
Comment (bootstraponline) [reply open_source_data/soseedygrpc/files.list]: There's some weird history about `soseedygrpc`

tl;dr it was only ever used on iOS and isn't actually used anymore by anyone
